.pre-commit-config.yaml

```diff
@@ -1,3 +1,16 @@
+repos:
+ - repo: https://github.com/keep-network/pre-commit-golang.git
+   rev: 4cc73f21101f9da1208719b02bbbe0a4c491497e
+   hooks:
+    - id: go-imports
+    - id: go-lint
+ - repo: https://github.com/tekwizely/pre-commit-golang.git
+   rev: v0.8.2
+   hooks:
+     # The `go-vet` hook is taken from the `tekwizely/pre-commit-golang.git`
+     # repo because `go-vet` from `keep-network/pre-commit-golang.git` doesn't
+     # work if Go files don't exist at the project root level.
+     - id: go-vet-pkg
```

Comment:
Non-blocking but is there an easy way to tweak `keep-network/pre-commit-golang.git` in the future and replace this part?

Reply:
Maybe, but I have not focused on that for now in order to not block here.

Reply:
For now, I will remove the `go-vet` hook completely as it still doesn't work well and cause problems on each commit. I'll add a backlog task for that and return here later.